### PR TITLE
Documentation Fixes

### PR DESCRIPTION
**Pull request by michaelschuff**
No logic changes so didn't feel the need to add a test bot

### CODE REVIEW

**Review by braindigitalis — COMMENTED** [review]
do we need all the other merges in here? I'm worried they might cause some issues and revert some changes?

File: include/dpp/message.h [hunk 1]
```diff
@@ -629,6 +629,7 @@ struct CoreExport sticker {
 	 * While discord still send this empty string value we will still have a field
 	 * here in the library.
 	 */
+	[[deprecated("Will always be an empty string")]]
```
[the remark this answers is not in the document]
Comment (braindigitalis) [reply include/dpp/message.h:632]: idk if I can remove them, but worst case scenario I'll just cherry pick out the correct commits 👍

[the remark this answers is not in the document]
Comment (michaelschuff) [reply include/dpp/message.h]: Alrighty thanks!

**Comment by braindigitalis** [comment]
looks good, thanks merged! if you're on the discord please DM me for a contributors role!

**Pull request merged**